src/contribute/release-guide.md

```diff
@@ -275,6 +275,8 @@ By default, the Javadoc will be generated in `target/site/apidocs/`. Let `${JAVA
 
 Please carefully review the generated Javadoc. Check for completeness and presence of all relevant packages and `package-info.java`; consider adding less relevant packages to the `excludePackageNames` configuration. The index page is generated at `${JAVADOC_ROOT}/index.html`.
 
+Create the python documentation using sphinx by running the script in the python sdk directory: `cd sdks/python && ./generate_pydocs.sh`. By default the pydocs are generated in `sdks/python/docs/_build`. Let `${PYDOC_ROOT}` be the absolute path to `_build`.
```

Comment:
Don't format commands to run inline, so people can copy-paste easily. Other things format as code inline.